subworkflows/local/call_variants.nf

```diff
@@ -10,11 +10,11 @@ include { CALL_VARIANTS_GATK } from './call_variants_gatk.nf'
 
 workflow CALL_VARIANTS {
     take:
-        ch_bam_bai     // channel (mandatory): [ val(meta), [ path(bam), path(bai) ] ]
-        ch_fasta       // channel (mandatory): [ path(fasta) ]
-        ch_fai         // channel (mandatory): [ path(fai) ]
-        ch_dict        // channel (mandatory): [ val(meta), path(dict) ]
-        variant_caller // string (mandatory)
+        ch_bam_bai     // channel: [ val(meta), [ path(bam), path(bai) ] ]
+        ch_fasta       // channel: [mandatory] [ val(meta), path(fasta) ]
+        ch_fai         // channel: [ val(meta),  path(fai) ]
+        ch_dict        // channel: [ val(meta), path(dict) ]
+        variant_caller // parameter: [ val(variant_caller) ]
```

Comment:
```suggestion
        variant_caller // parameter: variant_caller
```

Reply:
Instead, I will write the possibilities out, it will come on the next PR